pages/builds/environment_variables.md.erb

```diff
@@ -153,108 +157,117 @@ The following environment variables are automatically provided to every job, and
 
 You can define environment variables in your jobs in a few ways, depending on the nature of the value being set:
 
-* Pipeline settings — for values that are not secret.
+* Pipeline settings via the Buildkite dashboard — for values that are not secret.
 * [Build pipeline configuration](/docs/pipelines/command-step) — for values that are not secret.
-* An `environment` or `pre-command` [agent hook](/docs/agent/v2/hooks) — for values that are secret or agent-specific.
-
-## Environment variable precedence
+* An `environment` or `pre-command` [agent hook](/docs/agent/v3/hooks) — for values that are secret or agent-specific.
 
-You can set environment variables in lots of different places, and which ones take precedence can get a little confusing.
-There are many different levels at which environment variables are merged together. The following walkthrough and examples demonstrate the order in which variables are combined, as if you had set variables in every available place.
```

Comment:
Could we add back in a cut down version of this bit? I think some context is required before diving into such a complex thing.

Reply:
Yeah, I was wondering if that's what we should add back in. I'll give it a go!

Reply:
I tried and failed! 😓 I pushed the other changes though.

Reply:
Could I get your hand on maybe adding something back in?

Reply:
Sure! I was looking at it this morning, but got distracted by my sidebar reorganisation haha. I'll have a go now!